### PR TITLE
Shelf2

### PR DESCRIPTION
**Pull request by NoniiM**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -50,8 +50,4 @@ dependencies {
     implementation 'com.facebook.shimmer:shimmer:0.4.0'
 
 
-
-
-
 }
-
```
Comment (ThapeloNWU) [inline app/build.gradle:66]: delete file

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -19,76 +19,40 @@
...
-        <activity
-            android:name=".LoginActivity"
-            android:theme="@style/AppTheme.NoActionBar">
-
-        </activity>
-
+        <activity android:name=".on_shelf" />
     </application>
 
 </manifest>
```
Comment (ThapeloNWU) [inline app/src/main/AndroidManifest.xml:86]: delete file

File: app/src/main/java/com/drms/doneretail/MainActivity.java [hunk 3]
```diff
@@ -56,7 +36,9 @@ public void store(View view) {
     }
 
     public void shelf(View view) {
-        Toast.makeText(getApplicationContext(), "I work from SHELF", Toast.LENGTH_LONG).show();
+        Intent shelfIntent = new Intent(this, aisles.class);
+        startActivity(shelfIntent);
+        Toast.makeText(getApplicationContext(), "Welcome to SHELF Management", Toast.LENGTH_LONG).show();
     }
 
     public void pos(View view) {
```
Comment (ThapeloNWU) [inline app/src/main/java/com/drms/doneretail/MainActivity.java:126]: delete file

**Review by ThapeloNWU — CHANGES_REQUESTED** [review]

**Review by NoniiM — COMMENTED** [review]
check......

**Pull request merged**